packages/ui/src/common/ListItem.tsx

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (c) 2025 The Ontario Institute for Cancer Research. All rights reserved
+ *
+ *  This program and the accompanying materials are made available under the terms of
+ *  the GNU Affero General Public License v3.0. You should have received a copy of the
+ *  GNU Affero General Public License along with this program.
+ *   If not, see <http://www.gnu.org/licenses/>.
+ *
+ *  THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND ANY
+ *  EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES
+ *  OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT
+ *  SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT,
+ *  INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED
+ *  TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS;
+ *  OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER
+ *  IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
+ *  ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+/** @jsxImportSource @emotion/react */
+
+import { CSSProperties, ReactNode } from 'react';
+
+import { Theme } from '../theme';
+import { useThemeContext } from '../theme/ThemeContext';
+import Pill from './Pill';
+
+export interface ListItemProps {
+	children: ReactNode;
+}
+
+const getListItemStyles = (theme: Theme): CSSProperties => ({
+	fontSize: '13px',
+	fontWeight: 'normal',
+	lineHeight: '1',
```

Comment:
unless we're explicitly setting this for something, let's remove it. I've seen in a bunch of other PRs across the projects we tend to just copy pasta in Figma snippets, and it's overcomplicating the actual code. Things like font-family and line-height should be inherited for the most part